src/components/icons/Lock.tsx

```diff
@@ -0,0 +1,19 @@
+import { colorsV3 } from '@hedviginsurance/brand/dist'
+import * as React from 'react'
+
+export const Lock: React.FC = () => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 10 13"
+    width={14}
+    height={14}
```

Comment:
Are this one of the new icons? DIdn't ermir make them square and in 48 x 48

Reply:
Unsure, i got it from Zak's design. this is the padlock on the righthand side "when your old one expires"